mindjunkies/accounts/models.py

```
@@ -1,36 +1,22 @@
-        return self.create_user(username, email, password, **extra_fields)
-
-
 class User(AbstractUser):
+    """Custom User model with UUID primary key."""
+
     uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
 
     REQUIRED_FIELDS = ["email", "first_name", "last_name"]
-
+    is_teacher = models.BooleanField(default=False)
     objects = UserManager()
```

Ruff (0.8.2)

Undefined name `UserManager`

(F821)